lib/notifications/display.dart

```diff
@@ -238,4 +263,14 @@ class NotificationDisplayManager {
       page: MessageListPage(narrow: narrow)));
     return;
   }
+
+  static Future<Uint8List?> _fetchBitmap(Uri url) async {
+    try {
+      final resp = await http.get(url);
+      return resp.bodyBytes;
+    } catch (e) {
+      // TODO(log)
+      return null;
+    }
+  }
```

Comment:
I wonder how long this request could sit unresolved, and if we want to time out after some short-ish amount of time. Users will want to see their notifications as soon as we can reasonably show them, even if they're missing an avatar image, I think.

No need to block, this could be a TODO; we haven't implemented this in zulip-mobile.